src/Raven.Client/Documents/Session/Tokens/FalseToken.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Text;
+
+namespace Raven.Client.Documents.Session.Tokens
+{
+    public sealed class FalseToken : QueryToken
+    {
+        private FalseToken()
+        {
+        }
+
+        public static readonly FalseToken Instance = new FalseToken();
+
+        public override void WriteTo(StringBuilder writer)
+        {
+            writer
```

Comment:
There is actually no support for `false` in RQL

Reply:
Other ways to fix this? Or throw an exception. Having the opposite is a bit ackward, isn't it?

Reply:
I assigned the issue, we'll look int this.

Reply:
Thanks